components/mindstorms-ev3/src/main/eventflow/com/example/ev3/Demo.sbapp

```diff
@@ -0,0 +1,225 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>

```

Comment:
For both `.sbapp` files, can you format it (Cmd+L) so the arrows are drawn nicely?